### PR TITLE
Update SolarWinds integration to use utility

### PR DESCRIPTION
**Pull request by erikzaadi**
TUI FTW

@shaharke @dshush

### CODE REVIEW

File: _integrations/solarwinds.md [hunk 1]
```diff
@@ -5,43 +5,37 @@ draft: false
 type: System Monitoring
 ---
 
-#### Create system user account in SolarWinds
+#### Download the BigPanda SolarWinds integrator Utility
 
-If you already have a user account with system rights, proceed to the next step. If not, create a new account.
+`bigpanda-solarwinds-config` is a small command line application that will configure your BigPanda SolarWinds integration.
```
Comment (shaharke) [inline _integrations/solarwinds.md]: is it small?

**Comment by shaharke** [comment]
noice!

**Comment by dshush** [comment]
:+1:

**Comment by erikzaadi** [comment]
@rasienko please have a look

**Pull request merged**